ios/FlankerNativeComponents/GameManager.swift

```diff
@@ -110,70 +108,12 @@ class GameManager {
       switch type {
       case .fixations:
         startFixationsTimestamp = time
-
-        if
-          let startFeedbackTimestamp = startFeedbackTimestamp,
-          let endFeedbackTimestamp = endFeedbackTimestamp,
-          let gameParameters = gameParameters,
-          gameParameters.showFeedback {
-          let resultTime = (time - startFeedbackTimestamp) * 1000
-          let model = FlankerModel(rt: resultTime,
-                                   stimulus: "<div class=\"mindlogger-message correct\">\(responseText)</div>",
-                                   button_pressed: nil,
-                                   image_time: endFeedbackTimestamp * 1000,
-                                   correct: nil,
-                                   start_timestamp: 0,
-                                   tag: "feedback",
-                                   trial_index: countTest,
-                                   start_time: startFeedbackTimestamp * 1000,
-                                   response_touch_timestamp: 0)
-          delegate?.resultTest(avrgTime: nil, procentCorrect: nil, data: model, dataArray: nil, isShowResults: gameParameters.showResults, minAccuracy: gameParameters.minimumAccuracy)
-          resultManager.addStepData(data: model)
-        }
       case .trial:
-        startTrialTimestamp = time
```

Comment:
I removed startTrialTimestamp in the setEndTimeViewingImage method for case .trial to prevent overwriting the startTrialTimestamp that is set elsewhere in your code. Previously, setting startTrialTimestamp here was causing issues with calculating the average response time because it was unintentionally overwriting the timestamp.

By setting startTrialTimestamp directly where it's needed (in the setText() method), we ensure that the correct timestamp is used for response time calculations. Removing it from setEndTimeViewingImage avoids conflicts and maintains accurate timing.